src/styles/StyleUtils.js

```diff
@@ -37,7 +37,7 @@ const avatarBorderSizes = {
     [CONST.AVATAR_SIZE.SMALLER]: variables.componentBorderRadiusMedium,
     [CONST.AVATAR_SIZE.SMALL]: variables.componentBorderRadiusMedium,
     [CONST.AVATAR_SIZE.HEADER]: variables.componentBorderRadiusMedium,
-    [CONST.AVATAR_SIZE.DEFAULT]: variables.componentBorderRadiusNormal,
+    [CONST.AVATAR_SIZE.DEFAULT]: variables.componentBorderRadiusMedium,
```

Comment:
This will effect workspace avatars.
![Screenshot from 2023-05-12 16-48-30](https://github.com/Expensify/App/assets/16493223/600a8ab3-fb9d-448f-a7b5-b5d05e5a1d16)
![Screenshot from 2023-05-12 16-48-23](https://github.com/Expensify/App/assets/16493223/3f0a4c2e-f5a2-4b28-9630-236a9acadeee)

cc @shawnborton

Reply:
Hmm we should only be changing the border radius at the super small size when it is overlaid on top of a user avatar.

Reply:
Updated.